### PR TITLE
Initial README

### PR DESCRIPTION
**Pull request by ryangawei**
Add project proposal as the README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+
+As a shelter staff/administrator:
+
+- I want to help abandoned cats to find adopters by posting their information to public
+- I want analyze data related to the shelter, for example, how many cats are adopted/sheltered every month.
+
+As someone whose cat is missing:
+
+- I want to report my cat to lost & found and see if anyone can find it and report it to the shelter
+
```
Comment (AnnabelleAB) [inline README.md:22]: As an admin:

- I want to see the data statistics running for the above service for future decision-making.

**Review by AnnabelleAB — APPROVED** [review]
we can put the Homepage wireframe before the lost and found page, besides that, everything LGTM!

**Comment by ryangawei** [comment]
I think the homepage is already put before lost & found page, isn't it?

**Comment by AnnabelleAB** [comment]
> 

gotcha!! just some github issues when I check the" file changed", the order seems wrong there but good at the preview.

**Review by AnnabelleAB — APPROVED** [review]

**Pull request merged**